app/samples/src/main/java/com/sygic/samples/activities/SamplesActivity.kt

```diff
@@ -0,0 +1,73 @@
+package com.sygic.samples.activities
+
+import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.ActionBarDrawerToggle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.GravityCompat
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import com.sygic.samples.R
+import com.sygic.samples.viewmodels.SamplesActivityViewModel
+import com.sygic.ui.common.extensions.openUrl
+import kotlinx.android.synthetic.main.activity_samples.*
+
+class SamplesActivity : AppCompatActivity() {
+
+    private lateinit var samplesActivityViewModel: SamplesActivityViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContentView(R.layout.activity_samples)
+        setSupportActionBar(toolbar)
+        initDrawerToggle()
+
+        samplesActivityViewModel = ViewModelProviders.of(this).get(SamplesActivityViewModel::class.java).apply {
+            this.navigationItemListenerObservable.observe(
```

Comment:
tomuto prvemu observable nerozumiem. preco ked to emitne sa nastavi listener? co to ma robit?

Reply:
nahradím za data binding